[IMP] web: keep ui blocked until reload after module operations

SplashS opened the pull request:
Description of the issue/feature this PR addresses:
module operations (install, remove and update) ends with page reload.
during operation block ui usually performs cause operation takes time.
after operation and before page is reloaded ui is unblocked.
but reload also takes time because of asset rebuilding.

Current behavior before PR:
current behavior makes false feeling for the user that operation completes and interface ready for interaction.

Desired behavior after PR is merged:
after this commit ui will be blocked until page reloaded.







---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/odoo/pull/103226).

Julien00859 commented:
up @odoo/rd-framework-js

aab-odoo commented on addons/web/static/src/webclient/actions/action_service.js:
```diff
@@ -1277,6 +1277,9 @@ function makeActionManager(env) {
                     : { type: "ir.actions.act_window_close" };
+            if (action.type === "ir.actions.act_url" && action.target === "self" && action.params && action.params.block_ui) {
+                env.services.ui.block();
+            }
```
should be done in `_executeActURLAction`, and tested (url_action_tests.js) :D

SplashS commented:
@aab-odoo I rewrote code after your suggestions.
I added support for block_ui for reload client action also (in case of installing Website module).
But cause of legacy I have troubles with testing it. Could you give me some hints?
